velox/dwio/common/SelectiveColumnReader.h

```diff
@@ -733,7 +733,7 @@ class SelectiveColumnReader {
 
 template <>
 inline void SelectiveColumnReader::addValue(const std::string_view value) {
-  const uint64_t size = value.size();
+  const uint32_t size = value.size();
```

Comment:
It fix the [compile error](https://github.com/facebookincubator/velox/actions/runs/18688811569/job/53288733775) after change value type from `folly::StringPiece` to `std::string_view`. cc @pedroerp

Reply:
Revert it, change the memcpy() calls in `StringView` constructor to fix it.